deployment/media-insights-stack.yaml

```diff
@@ -104,11 +104,16 @@ Resources:
           import os
           def id_generator(size=6, chars=string.ascii_lowercase + string.digits):
             return "".join(random.choices(chars, k=size))
+          def init_mediaconvert_endpoint():
+            mediaconvert_client = boto3.client("mediaconvert", region_name=os.environ['AWS_REGION'])
+            response = mediaconvert_client.describe_endpoints()
+            mediaconvert_endpoint = response["Endpoints"][0]["Url"]
+            return mediaconvert_endpoint
           def init_system_table():
-            DYNAMO_CLIENT = boto3.resource("dynamodb")
+            dynamodb_client = boto3.resource("dynamodb")
```

Comment:
I made this variable lower case because I think we want to stay consistent with the pattern where only env vars are all caps.